.github/workflows/coverage.yaml

```diff
@@ -1,6 +1,11 @@
 name: coverage
 
-on: [push]
+on:
+  push:
```

Comment:
do we need `push` in addition to `pull_request_target`?

Reply:
This is a choice.

If you want someone who is developing to get feedback on this before they create a PR, then you could leave `push`. If you only want them to get feedback when they create a PR, then you can remove it.

Reply:
Let's remove it then, we can check coverage locally if needed. Would removing it also allow removing the `if (context.issue.number) {` in the comment step?